torch/_export/exported_program.py

```diff
@@ -173,6 +313,17 @@ def __str__(self) -> str:
     def graph(self):
         return self.graph_module.graph
 
+    def module(self, flat=True) -> Optional[torch.fx.GraphModule]:
```

Comment:
Let's start without having default value for flat?

Reply:
Yep good idea